assistant/v1.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2020.
+ * (C) Copyright IBM Corp. 2020.
```

Comment:
Make sure you go through all of the copyrights before merging

Reply:
Good catch